pkg/gameservers/controller.go

```diff
@@ -474,6 +477,35 @@ func (c *Controller) syncGameServerPortAllocationState(ctx context.Context, gs *
 	return gs, nil
 }
 
+// Allocate gameserver ports ( containerports ) for dynamic protocols such as TCPUDP
+// TODO: Going forward, this needs to be enhanced for other kinds of dynamic protocols
+func handleTCPUDPPortAllocationforStaticPortPolicy(gs *agonesv1.GameServer) {
```

Comment:
```suggestion
func assignStaticTCPUDPPorts(gs *agonesv1.GameServer) {
```
Other options from "assign": "apply", "create" ? "handle" feels generic, but we don't use it anywhere else either.

I feel like we aren't "allocating" a port, since it's static - we are just assigning existing static ports. WDYT?